readthedocs/oauth/services/bitbucket.py

```diff
@@ -120,13 +129,15 @@ def create_repository(self, fields, privacy=None, organization=None):
         """
         privacy = privacy or settings.DEFAULT_PRIVACY_LEVEL
         if any([
-                (privacy == 'private'),
-                (fields['is_private'] is False and privacy == 'public'),
+            (privacy == 'private'),
+            (fields['is_private'] is False and privacy == 'public'),
         ]):
             repo, _ = RemoteRepository.objects.get_or_create(
-                full_name=fields['full_name'],
-                account=self.account,
+                remote_id=fields['uuid'],
```

Comment:
We should probably document somewhere that the `remote_id` is for each VCS provider.